common/models/user.js

```diff
@@ -814,7 +814,7 @@ module.exports = function(User) {
         if (!ctx.res) {
           return next(new Error(g.f('The transport does not support HTTP redirects.')));
         }
-        ctx.res.location(ctx.args.redirect);
+        ctx.res.location(decodeURIComponent(ctx.args.redirect));
```

Comment:
I believe `ctx.args.redirect` should have been already url-decoded by strong-remoting, therefore this change is not necessary. Unless I am missing something?